documentation/modules/rn-2.2.adoc

```diff
@@ -5,25 +5,37 @@
 [id="rn-22_{context}"]
 = {project-full} 2.2
 
-You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+This release supports migration of virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+The precopy interval is now set as an environment variable on the forklift-controller pod's `controller` container.
```

Comment:
Passive voice. Please change to active.

Also, the technical side is not correct. Where did you get this "controller" container sentence?

The env var belongs to the forklift-controller DEPLOYMENT. Very different.